packages/frontend/src/hooks/echarts/styles/tooltipStyles.ts

```diff
@@ -0,0 +1,165 @@
+import { type MantineTheme } from '@mantine/core';
+
+/**
+ * Helper to convert style object to CSS string
+ * We need this because ECharts doesn't support style objects directly
+ */
+const stylesToString = (styles: Record<string, string | number>) =>
```

Comment:
Nit: but we could enforce better types here with CSSProperties (from either React or Mantine)
```suggestion
const stylesToString = (styles: CSSProperties) =>
```